root/tree/dataframe/wmass/runSignalAnalysis.py

```diff
@@ -19,6 +19,8 @@ def analysis(nthreads):
     else:
         ROOT.ROOT.DisableImplicitMT()
 
+    ROOT.gInterpreter.AddIncludePath('./')
```

Comment:
And what if to add ```inc``` here? And in next line ```#include "classes.h"```?

Reply:
Actually do you think we can't avoid it?

Reply:
i don't know how to make things work without these two lines (but feel free to advise!).

will change to `AddIncludePath('inc')`.

Reply:
It seems there are a few files that include headers as `inc/header.h` (namely, all headers in `inc` include each other like that), so it would require several changes to use `AddIncludePath('inc')`. Can I leave it like it is? Was it just an aesthetic thing or do you expect issues?

Reply:
Let's leave it like this..